data/anaconda.conf

```diff
@@ -266,6 +266,12 @@ hidden_spokes =
 # Run GUI installer in a decorated window.
 decorated_window = False
 
+# Should the UI allow to change the configured root?
```

Comment:
I think we should always talk about "root password" in this case as otherwise it's easy to confuse with install root/rootfs/root directory/etc.

Also it seems to me that we might want to clarify a bit more what "configured" means - maybe "configured via kickstart" or possibly "configured via the DBus API" ?

Reply:
Fixed.